### ISSUE TITLE
[Feature Request] Documentation on User Requirements "mods"/etc...

### ISSUE THREAD

**Issue by ElectricPrism** [Keriew/augustus#172]
Arch Linux User here:

It wasn't obvious to me that the /mods folder referenced in other bugs, documentation and error messages was infact literally just a copy/paste of the mods directory in the repo to my [...]/Caesar/S3/ directory.

I also noticed "app", "mp3", "maps", etc...

I'm still not clear on what "maps" is supposed to be or if there some data I am supposed to download or extract from somewhere to that directory.

I'm not sure what would go in "app" either.

It would be nice if this was explained in the documentation.

According to the AUR `augustus-game` the file list essentially unpacks the main binary to `/usr/bin/augustus-game`.
https://aur.archlinux.org/packages/augustus-game/

It would be nice if the game were to unpack to /opt/augustus-game/ and possibly by default install "app", "mp3", "mods", "maps" etc... to that directory -- then in the instance of Steam -- if they were to ever update or validate the game installation it wouldn't wipe out all these user-files.

Other than that, the game is playing nicely on 32" @ 4k with 300% UI Scale and 200% Cursor size _(It would be nice if cursor could boost to 300-400%)_

Thank you @ all for your amazing work making this really cool and amazing game mod.

**Comment by crudelios** [comment]
While there's already a pdf manual, I agree the main readme should be edited to provide better information regarding installation.

**Comment by crudelios** [comment]
Closing as duplicate of #182 (well, technically #182 is a duplicate of this, but the discussion there is more up-to-date).

TL;DR: I am considering the best way to do this.
